sharepoint/sharepoint-ps/sharepoint-online/Export-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,70 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+applicable: SharePoint Online
+online version: 
+schema: 2.0.0
+author: MachelleTranMSFT
+ms.author: mactra
+ms.reviewer: pullabhk
+---
+
+# Export-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+This commands exports/downloads the Data access governance report to the default path "C:\WINDOWS\system32\"

```

Comment:
>default path "C:\WINDOWS\system32\" [](http://example.com/codeflow?start=73&length=35)

is this correct? how is the default set to "C:\windows\system32"? What if the user has no C: drive, eg, uses D:\ ?

Reply:
also missing `.` at end of sentence